docs/great-houses/house-of-NFTs.md

```diff
@@ -36,78 +33,127 @@ When it comes to NFTs, you are only limited by your imagination!
 
-- **[MakersPlace](https://makersplace.com/)**  (MKT2) Curated platform. Editions  and one-of-ones, 'hides' the web3 aspect from artists and users to make it more attractive to traditional artists.
+## [MakersPlace](https://makersplace.com/)  (MKT2)
+Curated platform. Editions  and one-of-ones, 'hides' the web3 aspect from artists and users to make it more attractive to traditional artists.
  
-- **[Mintable](https://mintable.app/)** Provides a smart contract factory for artists to be able to create their own marketplace plus onsite tools like auctions and flexible licensing for digital assets. Has created an NFT DAO with $MINTS. 
+## [Mintable](https://mintable.app/)
+Provides a smart contract factory for artists to be able to create their own marketplace plus onsite tools like auctions and flexible licensing for digital assets. Has created an NFT DAO with $MINTS. 
 
-- **[Mintbase](https://mintbase.io/)** Provides a smart contract factory for artists to be able to create their own contracts, storefronts hosted on the Mintbase website. Be your own marketplace! Provides API access so storefronts can be integrated on artists' own websites. Porting to NEAR.
+## [Mintbase](https://mintbase.io/)
+Provides a smart contract factory for artists to be able to create their own contracts, storefronts hosted on the Mintbase website. Be your own marketplace! Provides API access so storefronts can be integrated on artists' own websites. Porting to NEAR.
 
-- **[Nifty Gateway](https://niftygateway.com/)** Curated platform. Introduced the concept of **drops** turning auctions into events.  Transactions happen off-chain, so can only be bought/sold on the site.  Each artwork gets its own token contract.
+## [Nifty Gateway](https://niftygateway.com/)
+Curated platform. Introduced the concept of **drops** turning auctions into events.  Transactions happen off-chain, so can only be bought/sold on the site.  Each artwork gets its own token contract.
 
-- **[OpenSea](https://opensea.io/)** Both the marketplace for all other marketplaces plus the ability to create your own smart contract and storefront. Interesting feature is that all metadata (including the image) can be changed at any time by the contract owner.
+## [OpenSea](https://opensea.io/)
+Both the marketplace for all other marketplaces plus the ability to create your own smart contract and storefront. Interesting feature is that all metadata (including the image) can be changed at any time by the contract owner.
 
-- **[Pixura](https://pixura.io/)** Provides a smart contract factory for artists to be able to create their own marketplace via the GUI, also offers developer tools to programmatically issue & manage tokens, run queries against your smart contracts with GraphQL, and search through token metadata with Elasticsearch.
+## [Pixura](https://pixura.io/)
```

Comment:
I think you need to rebase on master since these changes to the NFT page already got merged

Reply:
> I think you need to rebase on master since these changes to the NFT page already got merged

will do. thanks.

Reply:
I've rebased and sorted conflicts.